Reemit.Common.UnitTests/SharedReaderTests.cs

```diff
@@ -14,10 +18,200 @@ public void ReadBytes_Called_AdvancesRelativeOffsetAndOffset()
         const int readBytesCount = 2;
         
         // Act
-        sharedReader.ReadBytes(readBytesCount);
+        var actualBytes = sharedReader.ReadBytes(readBytesCount);
         
         // Assert
         Assert.Equal(sharedReaderOffset + readBytesCount, sharedReader.Offset);
         Assert.Equal(readBytesCount, sharedReader.RelativeOffset);
+        Assert.Equal(readBytesCount, actualBytes.Length);
+        Assert.NotNull(actualBytes);
```

Comment:
It seems like the `NotNull` assertion on `actualBytes` may be redundant. We have an equality assertion just below it comparing `actualBytes` to the expected result. This already asserts that the value is not null.

Reply:
Done.